.repo/sample-list-generator/src/constants.ts

```diff
@@ -0,0 +1,16 @@
+export type AvailableFolderTypes = (typeof AVAILABLE_FOLDERS)[number]['type'];
+
+// Define all available folders for samples
+export const AVAILABLE_FOLDERS = [
+  {
+    path: 'api-samples',
+    type: 'API_SAMPLE'
+  },
+  {
+    path: 'functional-samples',
+    type: 'FUNCTIONAL_SAMPLE'
+  }
+] as const;
```

Comment:
Once we resolve the above comment we could type this as `{ path: string, type: FolderType }[]` and we should get autocomplete in some IDEs :)